### PR TITLE
feature(private post)

### PR DESCRIPTION
**Pull request by TwanoO67**
add possibility to set a post as private with a password

### CODE REVIEW

File: internal/write.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
         <input type="text" name="blogPostTitle" class="blogPostTitle" placeholder="The post title" required value="<?php echo $post['title']; ?>" />
         <input type="text" name="blogPostAuthor" class="blogPostAuthor" placeholder="The post author" required value="<?php echo $post['author']; ?>" />
         <input type="url" name="blogPostImage" class="blogPostImage" placeholder="A featured image for this post" value="<?php echo $post['image']; ?>" />
+        <input type="text" name="blogPostPassword" class="blogPostTitle" placeholder="Password if you want to hide the post" required value="<?php echo $post['password']; ?>" />
```
Comment (johnroper100) [inline internal/write.php:14]: This can't be a required field, otherwise, looks good!

**Review by johnroper100 — CHANGES_REQUESTED** [review]
This looks good overall, however, the field in `write.php` for the password shouldn't be required.

File: internal/write.php (hunk 1 above)
Comment (TwanoO67) [reply internal/write.php]: yes, i forgot that one. it's fixed :)

**Review by johnroper100 — APPROVED** [review]
Awesome, thanks!

**Pull request merged**